api/v1/client/src/commonMain/kotlin/auth/AuthService.kt

```diff
@@ -45,6 +45,7 @@ class AuthService(
                 append("username", username)
                 append("password", password)
                 append("grant_type", "password")
+                append("scope", "offline_access")
```

Comment:
> This allows for having long-lived refresh tokens for the
CLI, while still having the refresh tokens for the web application
expire after a shorter period of time in case there is no user
activity.

You cannot be sure that this `AuthService` is only used from the CLI, as it is intentionally put into the `api` module.
Therefore, you cannot hard code the `offline_access` here.

The caller / user of `AuthService` should be able to set the scope when creating this service, or when actually generating the tokens.